### ISSUE TITLE
For video and audio Chat change the UserAgent

### ISSUE THREAD

**Issue by mxsteini** [ivelkov/teams-for-linux#52]
http://nelkinda.com/blog/microsoft-teams-on-linux/
This is really working. Just change the useragent.

**Comment by IsmaelMartinez** [comment]
you will then have an issue with mentions (they are annoyingly disappear after hitting space).

**Comment by brad** [comment]
Actually, it is possible to have both mentions and video working, @Laxen would you mind explaining how you did it?

See the discussion [here](https://github.com/ramboxapp/community-edition/pull/1916) for more details.

**Comment by Laxen** [comment]
Yes, the discussion @brad linked shows how to get both mentions and calls working for Rambox. The solution for teams-for-linux is similar, you just execute the JavaScript code in the "dom-ready" event of the window. I might put this up as a pull request on this project, but it's kind of pointless since it's abandoned and won't get merged.

However this doesn't seem to work when upgrading Electron to 3.0.0 or later. I've posted this as [an issue on a fork](https://github.com/JamieMagee/teams-for-linux/issues/8) that seems to be active.

**Comment by julian-alarcon** [comment]
Check this fork and star it! https://github.com/IsmaelMartinez/teams-for-linux It is currently supported!
These issue was fixed sometime ago on that fork!
